### DISCUSSION TITLE
Convoy of combines with different header sizes

### DISCUSSION THREAD

**Discussion by LitlleRaven** [Courseplay/courseplay#7715] [Q&A] [answered]
Hi, sorry if this question was asked, but i would like to know if it is possible to send 2 cobines on course but with different header size. I play mostly european maps and i like to keep my machinery as diverse as possible, including mentioned combines. Any answer is appreciated :)

**Comment by Tensuko** [comment, marked as answer]
Add working with together, devide that by the amount of combines.
Then generate a course with that width and the amount of tools on course.
The only thing left is to give them the offset to be spread even on the course.

**Reply by LitlleRaven** [reply]
Thanks, gonna try it!
